src/savegame_legacy.cpp

```diff
@@ -538,6 +538,7 @@ void player_activity::deserialize_legacy_type( int legacy_type, activity_id &des
         activity_id( "ACT_WAIT_NPC" ),
         activity_id( "ACT_CLEAR_RUBBLE" ),
         activity_id( "ACT_MEDITATE" ),
+        activity_id( "ACT_WASH" ),
```

Comment:
Please don't add anything to legacy things like this. They exist only to convert saves in old format to the current format.